src/picknik_ur_gazebo_scan_and_plan_config/package.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>picknik_ur_gazebo_scan_and_plan_config</name>
+  <version>2.11.0</version>
+
+  <description>Site configuration package for the UR5e in PickNik's space station world simulated by Gazebo.</description>
```

Comment:
Update description to mention this application

Reply:
Done.